README.md

```diff
@@ -1,2 +1,52 @@
-# design-cfps
-Repo to store Cilium CFP design docs
+This directory contains CFP design proposals for features impacting repos across the
+Cilium Github organization.
+
+# Purpose of CFPs
+
+The purpose of a Cilium Feature Proposal (CFP) is to allow community members to gain feedback
+on their designs from the Committers before the community member commits to
+executing on the design. By going through the design process, developers gain a
+high level of confidence that their designs are viable and will be
+accepted.
+
+NOTE: This process is not mandatory. Anyone can execute on their own design
+without going through this process and submit code to the respective repos.
+However, depending on the complexity of the design and how experienced the
+developer is within the community, they could greatly benefit from going through
+this design process first. The risk of not getting a design proposal approved
+is that a developer may not arrive at a viable architecture that the community will
+accept.
+
+# How to create CFPs
+
+To create a CFP, it is recommended to use the `CFP-003-template.md`
+file as an outline. The structure of this template is meant to provide a starting
+point for people. Feel free to edit and modify your outline to best fit your
+needs when creating a proposal. The title should be `CFP-###-subject.md` 
+where the number is the CFP issue number.
```

Comment:
How does a contributor know what the "CFP issue number" is?

Should contributors open a new issue first, or should they open a blank PR and then update it to pick a number? (How does k8s do this? :thinking: )

Reply:
k8s has you open an issue first, and uses that as the basis.

Reply:
Another question from me: What about design proposal for Cilium _org_ repos other than `cilium/cilium`? I guess the simplest answer here is that the process is something like this:
1. Create a CFP issue in the repo your design applies to. (ie, cilium/cilium, cilium/tetragon, etc)
2. Create the file in this repo, with a path of `cfps/<repo>/CFP-####-subject.md`.
3. Merging the file into this repo constitutes acceptance of the design (or something to that effect).

This is imposing this requirement on other projects (like Tetragon), but I suspect that's going to end up a win in the longer term.

Reply:
I've tried to capture this below. # 3 should be covered in "getting a designed approved" below. 

I don't think this is actually imposing a requirement on other projects because they could decided to do something else and we would end up with just a `cilium` folder